src/include/ipc/dai.h

```diff
@@ -71,7 +71,8 @@ struct sof_ipc_dai_config {
 
 	/* physical protocol and clocking */
 	uint16_t format;	/**< SOF_DAI_FMT_ */
-	uint16_t reserved16;	/**< alignment */
+	uint8_t group_id;	/**< DAI group ID, 0 means no group */
+	uint8_t reserved8;	/**< alignment */
```

Comment:
@lgirdwood A general question: should we mandate that ABI version is mentioned in doxygen for any new ABI struct fields? Now you basicly have to dig through firmware git logs to figure out whether a certain field was always there, or added later.

Reply:
@kv2019i yes, sounds like a good ide. Btw is this requirement in sof-docs ?
@slawblauciak care to add the comment - I think we are good to go if kernel is good  ?